dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -32,16 +32,37 @@ agentAddress udp6:161
                                                  #  system + hrSystem groups only
 view   systemonly  included   .1.3.6.1.2.1.1
 view   systemonly  included   .1.3.6.1.2.1.25.1
-
                                                  #  Default access to basic system info
-{% if snmp_rocommunities %}
-{% for community in snmp_rocommunities %}
+
+{% if SNMP_COMMUNITY is defined %}
+{% for community in SNMP_COMMUNITY %}
+{% if SNMP_COMMUNITY[community]['TYPE'] == 'RO' %}
 rocommunity {{ community }}
 rocommunity6 {{ community }}
+{% endif %}
+{% endfor %}
+{% endif %}
+
+{% if SNMP_COMMUNITY is defined %}
+{% for community in SNMP_COMMUNITY %}
+{% if SNMP_COMMUNITY[community]['TYPE'] == 'RW' %}
+rwcommunity {{ community }}
+rwcommunity6 {{ community }}
+{% endif %}
+{% endfor %}
+{% endif %}
+
+{% if SNMP_USER is defined %}
+{% for user in SNMP_USER %}
+{% if SNMP_USER[user]['SNMP_USER_PERMISSION'] == 'RO' %}
+rouser {{ user }} {{ SNMP_USER[user]['SNMP_USER_TYPE'] }}
+CreateUser {{ user }} {{ SNMP_USER[user]['SNMP_USER_AUTH_TYPE'] }} {{ SNMP_USER[user]['SNMP_USER_AUTH_PASSWORD'] }} {{  SNMP_USER[user]['SNMP_USER_ENCRYPTION_TYPE'] }} {{  SNMP_USER[user]['SNMP_USER_ENCRYPTION_PASSWORD'] }}
```

Comment:
>   [](start = 170, length = 2)

Use one blank, many times in this file

Reply:
I'm not sure what you mean by this.  Are you saying that after each "user" you'd like a blank line?

Reply:
Removed the extra spaces you noticed.  Thanks!